app/mailers/stem_applicant_confirmation_mailer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class StemApplicantConfirmationMailer < TransactionalEmailMailer
+  SUBJECT = 'VA Rogers STEM Scholarship, Application Confirmation'
+  GA_CAMPAIGN_NAME = 'stem_applicant_confirmation-10203-submission-notification'
+  GA_DOCUMENT_PATH = '/email/form'
```

Comment:
```suggestion
  GA_DOCUMENT_PATH = '/email/form'.freeze
```

I'll concede it's extra-paranoid, but freezing path strings means we're less susceptible to path-traversal vulnerabilities, should another layer of security fail.

Reply:
![image](https://user-images.githubusercontent.com/41960449/89573385-89ae8a80-d7f8-11ea-9470-9aef453aa3a3.png)

It looks like rubocop doesn't like that

Reply:
oh, it's already immutable, my bad